src/ccl_halomod.c

```diff
@@ -0,0 +1,360 @@
+#include <stdio.h>
```

Comment:
looks like this file has perhaps been deleted on master since you last merged master into the branch? should be resolved by merging master in.

Reply:
will do

Reply:
now removed